terms/evaluationmethods_ae.bib

```diff
@@ -0,0 +1,247 @@
+}
+
+@book{Hyndman2018,
+    title = {Forecasting: Principles and Practice},
+    author = {Hyndman, Rob J and Athanasopoulos, George},
+    year = {2018},
+    publisher = {OTexts},
+}
+
+@article{Likert1932,
+    title = {A technique for the measurement of attitudes.},
+    author = {Likert, Rensis},
+    journal = {Archives of psychology},
+    year = {1932},
+}
+
+@article{Alonso2008,
+    author = {Alonso, Omar and Rose, Daniel E. and Stewart, Benjamin},
+    title = {Crowdsourcing for Relevance Evaluation},
+    journal = {SIGIR | Special Interest Group on Information Retrieval},
+    issue_date = {December 2008},
+    volume = {42},
+    number = {2},
+    month = {nov},
+    year = {2008},
+    pages = {9--15},
+    publisher = {ACM},
+}
+
+@book{Urdan2011,
+    title = {Statistics in plain English},
+    author = {Urdan, Timothy C},
+    year = {2011},
+    publisher = {Routledge},
+}
+
+@Book{Efron93,
+    Title = {An Introduction to the Bootstrap},
+    Author = {Bradley Efron and Robert J. Tibshirani},
+    Publisher = {Chapman \& Hall/CRC},
+    Year = {1993},
+    Number = {57},
+    Series = {Monographs on Statistics and Applied Probability},
+}
+
+@book{Harrell2015,
+    author = {Harrell, Frank E},
+    title = {Regression modeling strategies},
+    year = {2015},
+    publisher = {Springer},
+}
+
+@article{Mcnemar1947,
+    title = {Note on the sampling error of the difference between correlated proportions or percentages},
+    author = {McNemar, Quinn},
+    journal = {Psychometrika},
+    volume = {12},
+    number = {2},
+    pages = {153--157},
+    year = {1947},
+    publisher = {Springer},
+}
+
+@book{Friedman2001,
+    title={The elements of statistical learning},
+    author={Friedman, Jerome and Hastie, Trevor and Tibshirani, Robert},
+    year={2001},
+    publisher={Springer Series in Statistics}
+}
+
+@article{Hand2006,
+    title = {Classifier technology and the illusion of progress},
+    author = {Hand, David J},
+    journal = {Statistical science},
+    pages = {1--14},
+    year = {2006},
+    publisher = {JSTOR},
+}
+
+@article{Drummond2010,
+    title = {{Warning: statistical benchmarking is addictive. Kicking the habit in machine learning}},
+    author = {Drummond, Chris and Japkowicz, Nathalie},
+    journal = {Journal of Experimental \& Theoretical Artificial Intelligence},
+    volume = {22},
+    number = {1},
+    pages = {67--80},
+    year = {2010},
+    publisher = {Taylor \& Francis},
+}
+
+@inproceedings{Belz2008,
+    author = {Belz, Anja and Gatt, Albert},
+    title = {Intrinsic vs. Extrinsic Evaluation Measures for Referring Expression Generation},
+    booktitle = {Proceedings of the 46th Annual Meeting of the Association for Computational Linguistics on Human Language Technologies: Short Papers},
+    year = {2008},
+    location = {Columbus, Ohio},
+    pages = {197--200},
+    numpages = {4},
+    publisher = {Association for Computational Linguistics},
+}
+
+@misc{Dua2017,
```

Comment:
what is this a book?  a technical report?  Please make sure you select correct type of publication

Reply:
If you mean Dua2017, it is the UCI website cited [as per their policy](https://archive.ics.uci.edu/ml/citation_policy.html).

Reply:
Thanks for the correction, I'll take it as is.